### PR TITLE
Commerce packaging fixes

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | n/a
| Versions      | 3.x, master

Fixes mistakes in marking shop features' belonging to product editions, and clears up some sections in "Shop features".

### CODE REVIEW

File: docs/guide/customers/customers.md [hunk 1]
```diff
@@ -37,7 +23,7 @@ The ERP system provides:
 
 ## Customer data model
 
-[[= product_name_com =]] uses the UBL standard to model customer data. The most important type is the Party which describes an address. 
+[[= product_name =]] uses the UBL standard to model customer data. The most important type is the Party which describes an address.
```
Comment (dabrt) [inline docs/guide/customers/customers.md:26]:
```suggestion
[[= product_name =]] uses the UBL standard to model customer data. 
The most important data type is Party.
Party describes an address. 
```

File: docs/guide/forms/form_api/dataprocessors.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ The data processor sets the following fields in the User Content item:
 - `ses_hastopay_vat`
 - `ses_display_vat`
 
-[[= product_name_com =]] provides a standard event for this handler which adds a prefix to the login name of the user.
+[[= product_name =]] provides a standard event for this handler which adds a prefix to the login name of the user.
```
Comment (dabrt) [inline docs/guide/forms/form_api/dataprocessors.md:72]:
```suggestion
[[= product_name =]] provides a standard event for this handler, which adds a prefix to the login name of the user.
```

File: docs/guide/forms/forms.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+[[= product_name =]] offers a unified way of handling all one-page forms.
 
 One-page forms have several common criteria:
 
 - They are displayed on a single page.
 - After submitting some processes are executed in the backend.
 - After server response the user sees a confirmation page with a success or error message.
 
-[[= product_name_com =]] uses [Symfony forms](http://symfony.com/doc/3.4/forms.html) as a part of the solution.
+[[= product_name =]] uses [Symfony forms](http://symfony.com/doc/3.4/forms.html) as a part of the solution.
```
Comment (dabrt) [inline docs/guide/forms/forms.md:11]:
```suggestion
[[= product_name =]] uses [Symfony forms](http://symfony.com/doc/3.4/forms.html) as part of the solution.
```

File: docs/guide/order_history/orderhistory_configuration.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ Order history uses semantic configuration, so it only exposes parameters that ar
 
 However it is possible to [override this configuration per SiteAccess](overriding_semantic_configuration.md).
 When an event is thrown, before the configuration is used, you can implement a listener that changes this configuration.
-[[= product_name_com =]] uses this event to display [local orders](order_history_features/orderhistory_local_orders.md).
+[[= product_name =]] uses this event to display [local orders](order_history_features/orderhistory_local_orders.md).
 See the [Overriding semantic configuration](overriding_semantic_configuration.md) to find out how to implement a new configuration listener.
```
Comment (dabrt) [inline docs/guide/order_history/orderhistory_configuration.md:8]:
```suggestion
See [`Overriding semantic configuration](overriding_semantic_configuration.md) to find out how to implement a new configuration listener.
```

File: docs/guide/shop_features/catalog.md [hunk 5]
```diff
@@ -1,28 +1,28 @@
...
+[Data providers](../data_providers/data_providers.md) define where product information is stored.
 
 You can use one of the following data providers: the Repository data provider and the eContent data provider.
 
 ### Repository data provider
 
 When you use the Repository data provider, each product is a Content item, stored in the Repository.
 
-Use this data provider when you have no PIM system and a limited number of products (up to 20.000).
+Use this data provider when you have no ERP system and a limited number of products (up to 20.000).
```
Comment (dabrt) [inline docs/guide/shop_features/catalog.md:17]:
```suggestion
Use this data provider when you have no ERP system and a limited number of products (up to 20 000).
```

File: docs/guide/shop_features/catalog.md [hunk 6]
```diff
@@ -1,28 +1,28 @@
...
 
 You can create and edit all products directly in the Back Office.
 
 Importing products when using this data provider is time-consuming.
 
 ### eContent data provider
 
-Use the eContent data provider when you have a PIM system or an ERP which provides product information.
+Use the eContent data provider when you have an ERP system which provides product information.
```
Comment (dabrt) [inline docs/guide/shop_features/catalog.md:25]:
```suggestion
Use the eContent data provider when you have an ERP system that provides product information.
```

File: docs/guide/shop_features/catalog.md [hunk 7]
```diff
@@ -38,8 +38,10 @@ to learn how to change the data provider.
 
 ## Product categories
 
-![product catalog](img/product_catalog_2.png)
+![Product category](img/product_catalog_2.png)
 
-A category represents a product group. [[= product_name_com =]] enables showing a product category page using different layouts (by default sub-categories only, products only or both on the entry page of a category). In the Back Office you can also configure if bestsellers of the group should be displayed.
+A category represents a product group.
+You can show a product category page using different layouts (sub-categories only, products only or both on the entry page of a category).
```
Comment (dabrt) [inline docs/guide/shop_features/catalog.md:44]:
```suggestion
You can show a product category page by using different layouts (sub-categories only, products only or both on the entry page of a category).
```
Does "on the entry page of a category" belong outside of the parenthesis? Could it be "different layouts on the entry page of a category"?

File: docs/guide/shop_features/catalog.md [hunk 8]
```diff
@@ -38,8 +38,10 @@ to learn how to change the data provider.
...
+In the Back Office, you can also configure if bestsellers of the group should be displayed.
```
Comment (dabrt) [inline docs/guide/shop_features/catalog.md:45]:
```suggestion
In the Back Office, you can also configure whether bestsellers of the group should be displayed.
```

File: docs/guide/shop_features/payment_and_shipping.md [hunk 9]
```diff
@@ -1,13 +1,8 @@
-# Payment and shipping
+# Payment and shipping [[% include 'snippets/commerce_badge.md' %]]
 
-[[= product_name =]] offers the following payment methods:
+Payment is based on a standard Symfony bundle. By default, the invoice payment is available.
```
Comment (dabrt) [inline docs/guide/shop_features/payment_and_shipping.md:3]:
```suggestion
Payment is based on a standard Symfony bundle. 
By default, the invoice payment is available.
```

File: docs/guide/shop_features/pricing.md [hunk 10]
```diff
@@ -9,13 +9,14 @@ In addition, prices can vary per customer group.
 
 ## Currencies
 
-The currency is configured per country in the [configuration settings](../shop_configuration/shop_configuration.md#currencies-used-per-country).
+The currency is configured per country in the [configuration settings](../shop_configuration.md#price).
```
Comment (dabrt) [inline docs/guide/shop_features/pricing.md:12]:
```suggestion
You configure the currency per country in the [configuration settings](../shop_configuration.md#price).
```

File: docs/guide/shop_features/pricing.md [hunk 11]
```diff
@@ -9,13 +9,14 @@ In addition, prices can vary per customer group.
...
 
-If a product has a price for a currency this price is displayed in the shop.
+If a product has a price for a currency, this price is displayed in the shop.
 
-If there is no price set in a shop for the given currency, [[= product_name_com =]] offers two options that can be set in the shop configuration:
+If there is no price set in a shop for the given currency, you can set one of two options in the shop configuration:
 
-- Calculate the price for the requested currency using the base price defined in the product (using the base currency setup for the installation) and an exchange rate defined in the configuration
+- Calculate the price for the requested currency using the base price defined in the product
```
Comment (dabrt) [inline docs/guide/shop_features/pricing.md:18]:
```suggestion
- Calculate the price for the requested currency based on the base price defined in the product
```

File: docs/guide/shop_features/product_management.md [hunk 12]
```diff
@@ -0,0 +1,24 @@
+# Product management
+
+## Product
+
+[[= product_name =]] offers a built-in [Product Content Type](../catalog/product.md).
+
+Any content of this type can be automatically used for all shop functionalities,
+such as adding to basket, price calculation and ordering,
+and can be embedded on the front end with quick ordering buttons.
```
Comment (dabrt) [inline docs/guide/shop_features/product_management.md:9]:
```suggestion
and can be embedded on the front end with quick ordering buttons.
```
Does "with quick ordering buttons" mean "together with..."?

File: docs/guide/shop_features/product_management.md [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+
+## Product variants
+
+A product can have different variants, corresponding for example to different colors or sizes.
```
Comment (dabrt) [inline docs/guide/shop_features/product_management.md:13]:
```suggestion
A product can have different variants that correspond, for example, to different colors or sizes.
```

File: docs/guide/shop_features/product_management.md [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+
+You can select and preview variants on the product's page.
+Variants can have one or two levels.
+When you choose a first attribute, the shop narrows down the options for the second level.
```
Comment (dabrt) [inline docs/guide/shop_features/product_management.md:17]:
```suggestion
When you choose the first attribute, the shop narrows down the options for the second level.
```
Could it be:
"When you choose an option for the attribute at the first level, the shop narrows down the options for the second level."
?

File: docs/guide/shop_features/product_management.md [hunk 15]
```diff
@@ -0,0 +1,24 @@
...
+
+![Product detail](img/product_detail.png)
+
+## Product type
+
+A product type represents a collection of very similar products that differ only in some characteristics.
+It is used to show a list of products in a tabular way, every product can be added to the basket directly from the overview page.
```
Comment (dabrt) [inline docs/guide/shop_features/product_management.md:24]:
```suggestion
It is used to show a list of products in a tabular way, and every product can be added to the basket directly from the overview page. 
```

File: docs/guide/shop_features/user_management.md [hunk 16]
```diff
@@ -1,6 +1,6 @@
-# User management [[% include 'snippets/commerce_badge.md' %]]
+# User management
 
-[[= product_name_com =]] by default adds the Shop users User Group, containing subgroups Business customers and Private customers.
+[[= product_name =]] by default adds the Shop users User Group, containing subgroups Business customers and Private customers.
```
Comment (dabrt) [inline docs/guide/shop_features/user_management.md:3]:
```suggestion
By default, [[= product_name =]] adds the Shop users User Group, which contains the Business customers and Private customers subgroups.
```

**Review by dabrt — APPROVED** [review]

File: docs/guide/shop_features/catalog.md (hunk 7 above)
Comment (DominikaK) [inline docs/guide/shop_features/catalog.md:44]:
```suggestion
You can show a product category page by using different layouts (sub-categories only, products only or both) on the entry page of a category.
```

File: docs/guide/shop_features/product_management.md (hunk 12 above)
Comment (DominikaK) [inline docs/guide/shop_features/product_management.md:9]:
```suggestion
and can be embedded on the front end together with quick ordering buttons.
```

File: docs/guide/order_history/orderhistory_configuration.md [hunk 17]
```diff
@@ -4,8 +4,8 @@ Order history uses semantic configuration, so it only exposes parameters that ar
 
 However it is possible to [override this configuration per SiteAccess](overriding_semantic_configuration.md).
 When an event is thrown, before the configuration is used, you can implement a listener that changes this configuration.
-[[= product_name_com =]] uses this event to display [local orders](order_history_features/orderhistory_local_orders.md).
-See the [Overriding semantic configuration](overriding_semantic_configuration.md) to find out how to implement a new configuration listener.
+[[= product_name =]] uses this event to display [local orders](order_history_features/orderhistory_local_orders.md).
+See [`Overriding semantic configuration](overriding_semantic_configuration.md) to find out how to implement a new configuration listener.
```
Comment (DominikaK) [inline docs/guide/order_history/orderhistory_configuration.md:8]:
```suggestion
See [Overriding semantic configuration](overriding_semantic_configuration.md) to find out how to implement a new configuration listener.
```

**Pull request merged**